content/post/projects/flutter-bond/index.ar.md

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
That should be in the `software` folder.

Reply:
Should I move the flutter-bond folder to the software folder?

Reply:
Yes, please.